README.gcp.md

```diff
@@ -0,0 +1,127 @@
+# DDS - Build, test, and deploy using the Google Cloup Platform (GCP)
+
+For project info please see README.md
+
+# Connect to the cloud shell
+
+Using your favorite web browser, open a cloud shell terminal at <https://ssh.cloud.google.com/cloudshell>.
+You'll need a free Google account. These days, most people have one already.
+
+# Build and install the C++ library
+
+This repository contains the hackathon's fork of Bo Haglund's double-dummy solver code.
+
+```
+gcloud config set project online-bridge-hackathon-2020
+mkdir ~/bridge-hackathon/
+cd ~/bridge-hackathon/
+git clone https://github.com/suokko/dds.git
+cd dds/
+mkdir .build
+cd .build/
+cmake ..
+make
+```
+
+Optional, to make sure the library built correctly.
+It will take about six minutes to complete.
+```
+make check
+```
+
+Must be copied. A hard link is not possible and a symbolic link won't work:
+```
+sudo cp -f src/libdds.so.2 /usr/lib/
+```
+
+The cloud shell clears out /usr/lib/ every so often, so run this to add the *cp* command to your *.bashrc* file:
+```
+echo 'sudo cp $HOME/bridge-hackathon/dds/.build/src/libdds.so.2 /usr/lib/' >> ~/.bashrc
+```
+
+# Build and test the DDS webservice wrapper for the double-dummy solver library
+
+Note that the C++ project is named **dds** and the webservice project is named **DDS**. Yes, this could be confusing - we plan to rename one or both.
+
+```
+cd ~/bridge-hackathon/
+git clone https://github.com/online-bridge-hackathon/DDS.git
+cd DDS/
+make run_local_tests
+```
+
+# Install and run the DDS webservice in a local server
+
+
+```
+pip3 install Flask
+pip3 install flask-restful
+pip3 install flask_cors
```

Comment:
IMO options using requirements.txt for install would be better. Then we have to just maintain one place for dependencies.
`pip3 install -r requirements.txt`

Reply:
Quite right. I'll update the doc.

Reply:
Running the command in the cloud shell gets me this:

```
mssql-cli 1.0.0 has requirement click<7.1,>=4.1, but you'll have click 7.1.2 which is incompatible.
mssql-cli 1.0.0 has requirement prompt-toolkit<2.1.0,>=2.0.0, but you'll have prompt-toolkit 3.0.5 which is incompatible.
Installing collected packages: click
Successfully installed click-7.1.2
```

Per <https://pypi.org/project/click/>, "Click is a Python package for creating beautiful command line interfaces". Does anyone know why we require it? I see that @Brewing1 added it.

Reply:
Not me